docs/rules/object-key-newlines.md

```diff
@@ -0,0 +1,68 @@
+# Enforce placing object keys on separate lines (object-key-newlines)
+
+While formatting preferences are very personal, a number of style guides require that
```

Comment:
@pedrottimark I think I now finally understand your point about property vs. key, and yes, that example you gave would be flagged as incorrect. Agreed that property instead of key would be better. Shall I rename the rule to `object-property-newline` then?